### ISSUE TITLE
possible mistake in matchingR/vignettes/matchingR-intro.Rmd

### ISSUE THREAD

**Issue by HakanHolmberg** [jtilly/matchingR#23]
Hi again, I may be completely wrong but I think that in the mens payoff matrix uM Rows corresponds to men and not columns as it says now.

the same holds for womens payoff matrix uW

If I am correct there is two problems with the preference order matrix prefM 

1) rows corresponds to men as in uM
2) and if row 1 corresponds to the preference order of man 1, then the preference order is not proper since it contains two 3's

thanks for a great package otherwise, best regards
Haakan

**Comment by njanetos** [comment]
The labels were switched for some of the tables. Everything is in column order. Thank you for finding this.

**Comment by HakanHolmberg** [comment]
I thin that 

prefM = matrix(c(1, 3, 3,
                 2, 1, 2,
                 3, 2, 1), nrow = 3, ncol = 3, byrow = TRUE)

should be

prefM = matrix(c(1, 2, 3,
                 2, 3, 2,
                 3, 1, 1), nrow = 3, ncol = 3, byrow = TRUE)

**Comment by njanetos** [comment]
If I understand you correctly, you are running the following:

```
prefM = matrix(c(1, 2, 3,
2, 3, 2,
3, 1, 1), nrow = 3, ncol = 3, byrow = TRUE)

prefW = matrix(c(3, 1, 3,
2, 3, 2,
1, 2, 1), nrow = 3, ncol = 3, byrow = TRUE)

matching <- one2one(proposerPref = prefM, reviewerPref = prefW)
```
When I run this, it returns
```
$proposals
     [,1]
[1,]    1
[2,]    2
[3,]    3

$engagements
     [,1]
[1,]    1
[2,]    2
[3,]    3

$single.proposers
numeric(0)

$single.reviewers
numeric(0)
```
 Did you maybe turn on row-order preferences with `set.row.major()`?

**Comment by jtilly** [comment]
Hakam, we released a new version (1.1) of this package on CRAN about a week ago.  Are you using the latest version? The vignette code corresponds to this new version of the package. 

You can check which version of matchingR you have installed by typing
```{r}
packageVersion("matchingR")
```
into your R session.

If this isn't 1.1, you should install the latest version:
```{r}
install.packages("matchingR")
```

**Comment by HakanHolmberg** [comment]
> packageVersion("matchingR")
[1] ‘1.0.1’

so you are absolutely correct, thank you so much for you quick response. I installed the latest version and now I get the ssame result as njanetos..
